CHANGELOG.md

```diff
@@ -15,6 +15,18 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 - Expression builder plugin.
 - 107+ JIRA tasks :-)
 
+## [1.4.1] - 2018-04-03
+### Added
+- Start to use fastdom to increase rendering perfomance.
+- The latest version of vscroll is used.
```

Comment:
Will persistence plugin be added to the release?

Reply:
this is ng1 version :)